cli/flags.go

```diff
@@ -0,0 +1,697 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//nolint:wrapcheck // These functions intentionally just wrap flag.Flag.
+package cli
```

Comment:
It'd be great if we can have an example in which a "leaf" command has a couple of flags. It would help to illustrate how to use the flags with a command. (I had to read the JVS change to get a grip on it)